pyartcd/tests/pipelines/test_update_golang.py

```
@@ -1500,5 +1508,259 @@ async def test_rebase_and_build_konflux(self, mock_cmd_assert, mock_konflux_db):
+class TestShouldSignGolangRpm(unittest.TestCase):
+    """Test the should_sign_golang_rpm method"""
+
+    @patch("pyartcd.pipelines.update_golang.KonfluxDb")
+    @patch("pyartcd.pipelines.update_golang.get_github_client_for_org")
+    def test_returns_true_when_config_set(self, mock_get_github_client, mock_konflux_db):
+        mock_repo = Mock()
+        mock_repo.get_contents.return_value = Mock(decoded_content=b"sign_golang_rpm: true\n")
+        mock_get_github_client.return_value.get_repo.return_value = mock_repo
+
+        pipeline = UpdateGolangPipeline(
+            runtime=Mock(dry_run=False, working_dir=Path("/tmp/working")),
```

Ruff (0.15.17)

[error] Probable insecure usage of temporary file or directory: "/tmp/working"

(S108)